firebase-functions/functions/src/operations/RoundOperations.ts

```diff
@@ -35,3 +46,24 @@ function getResult(player: QueryDocumentSnapshot<DocumentData>, allPlayers: Quer
       ? Result.Lost
       : Result.Won;
 }
+
+async function getRoom(roomId: string): Promise<Room> {
+  const roomDoc = await admin.firestore().collection(collections.rooms).doc(roomId).get();
+  return roomDoc.data() as Room;
+}
+
+function defineLastOneActive(playerChoices: PlayerChoice[], typeOfGame?: GameType): string | undefined {
```

Comment:
Isn't this very similar to what you mentioned in my PR? We are calculating this `lastOneActive` in `playerOperations`. Maybe we can return it there and pass it into this function here.  
Also, if I am wrong, this method name is confusing. Why not `determineLastOneActive` (which is the function name in said file :) )?

Reply:
Not the same, you were finding the result for each one, I am using the result to find who is the `lastOneActive`, I renamed it to `findLastOneActive` for now because I am not determining (or defining) I just want to find it. I am not sure if I am clear, but happy to discuss it more.

The story related to my comments is still in progress, I think we can reuse some of your functions but I didn't want to confuse us (myself) with a lot of changes. now after this is approved I think one of us can give an attempt to refactor the firebase functions,  especially that we gonna remove the lastOneActive from the room.

Reply:
![image](https://user-images.githubusercontent.com/17160319/215165765-f37abf5e-9888-4342-a050-6bae8e106446.png)

Alright, I still think it could just be passed in, but WHATEVS....